sensor/common/sensor/central_sender_impl.go

```diff
@@ -96,6 +112,7 @@ func (s *centralSenderImpl) send(stream central.SensorService_CommunicateClient,
 			s.stopper.Flow().StopWithError(stream.Context().Err())
 			return
 		}
+		metrics.DecResponsesChannelSize()
```

Comment:
A thought: If this function exits, we must make sure that the value is set to 0. Is the gauge value preserved during restarts? I think prometheus will keep the last value until next update arrives. So we must set it to 0 on start or 0 on exit (or both).

Alternatively, we could have one counter vector with two operations - push & pop - and increase those two (similarly to the way we do for the PI and netflow queues)

Reply:
> Alternatively, we could have one counter vector with two operations - push & pop - and increase those two (similarly to the way we do for the PI and netflow queues)

I like that solution.